### ISSUE TITLE
Getting E429

### ISSUE THREAD

**Issue by surya-purohit** [eslint/eslint#12929]
**Tell us about your environment**

* **ESLint Version:** latest
* **Node Version:** 8.9.4
* **npm Version:** 5.6.0

**What parser (default, Babel-ESLint, etc.) are you using?**
`babel-eslint`
**Please show your full configuration:**
<details>
<summary>Configuration</summary>

```js
{
  parser: "babel-eslint",
  parserOptions: {
    ecmaVersion: 6,
    sourceType: "module",
    allowImportExportEverywhere: true
  },
  extends: ["prettier", "airbnb-base"],
  rules: {
    "max-len": ["warn", { code: 160 }],
    "global-require": 0,
    "no-console": 0,
    "no-continue": 0,
  },
};
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
npm i eslint --save-dev
```

```bash

```

**What did you expect to happen?**

**What actually happened? Please include the actual, raw output from ESLint.**
```npm ERR! code E429
npm ERR! 429 Too Many Requests```

**Are you willing to submit a pull request to fix this bug?**

**Comment by alopix** [comment]
This is an ongoing issue with NPM right now and does not have to do with eslint or any specific package.

**Comment by surya-purohit** [comment]
It's an issue with NPM itself
